lib/semian/redis.rb

```diff
@@ -13,6 +13,7 @@ def initialize(semian_identifier, *args)
   end
 
   class OutOfMemoryError < Redis::CommandError
+    include ::Semian::AdapterError
```

Comment:
I think it would be better to do:

`OutOfMemoryError.include(::Semian::AdapterError)`

This way we're sure the class is already loaded, and we don't mess things up by defining it ourselves.

Reply:
Scratch that. I forgot this isn't an upstream exception class.